testing/001/indexpage.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="">
+  <project>
+    <projectname>My Project    </projectname>
```

Comment:
@albert-github any idea where these extra spaces at the end are coming from? Same question for projectbrief. And maybe better to only write the project info if it is different from the default.

Reply:
Yes in the #153 version there was the statement  `"      </projectname>"` hence the extra spaces, with #8998 this has been fixed as well (lines 155 and 159 of xmlgen.cpp).